README.md

```diff
@@ -185,3 +170,9 @@ If you want to export only the reports, then it would need to look like this:
 **Script Failure:**
 
 If the script fails without dropping the bk_* (where * is the name of the hpr file or its .bk sql server backup database) database, then you will need to do it manually. Using SQL Server Management Studio you can DELETE the bk_* database and using Windows explorer you can delete the temp folder that contains the unzipped contents of the HPR file.
+
+## Contact
+
+Issues can be reported through the repository on Github (https://github.com/nhrap-hazus/export)
+
+For questions contact fema-hazus-support@fema.dhs.gov
```

Comment:
Minor punctuation nit: throw a semicolon after questions.... it should read:

 _"For questions: contact fema-hazus-support@fema.dhs.gov"_

Reply:
Added colon